webapp/src/js/components/containers/FindGeneByNameDesc.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import PureRenderMixin from 'mixins/PureRenderMixin';
+import FluxMixin from 'mixins/FluxMixin';
+
+// Panoptes
+import GeneSearchResultsList from 'panoptes/GeneSearchResultsList';
+
+// Material UI
+import TextField from 'material-ui/TextField';
+
+
+let FindGeneByNameDesc = React.createClass({
+  mixins: [
+    PureRenderMixin,
+    FluxMixin
+  ],
+
+  title() {
+    return this.props.title;
+  },
+
+  getInitialState() {
+    return {
+      search: ''
+    };
+  },
+
+  handleSearchChange(event) {
+    this.setState({'search': event.target.value});
```

Comment:
Because search is state we loose it if the popup is maximised or vice-versa. I could be in props.